www/package.json

```diff
@@ -11,12 +11,20 @@
     "sync-stripe": "node utils/stripe/sync.ts"
   },
   "dependencies": {
+    "@radix-ui/react-icons": "^1.3.0",
+    "@radix-ui/react-label": "^2.1.0",
+    "@radix-ui/react-slot": "^1.1.0",
+    "@radix-ui/react-tabs": "^1.1.1",
     "@sentry/nextjs": "^8.34.0",
     "@stripe/stripe-js": "^4.8.0",
     "@supabase/ssr": "^0.5.1",
     "@supabase/supabase-js": "^2.45.4",
+    "class-variance-authority": "^0.7.0",
+    "clsx": "^2.1.1",
     "katex": "^0.16.11",
+    "lucide-react": "^0.452.0",
```

Comment:
We shouldn't be using `lucide-react` and `react-icons` basically let's try to only use one icon library